detox/test/e2e/06.device.test.js

```diff
@@ -84,4 +84,35 @@ describe('Device', () => {
     await device.pressBack();
     await expect(element(by.text('Back pressed !'))).toBeVisible();
   });
+
+  it(':android: tap on screen', async () => {
+    await device.launchApp({ newInstance: true });
+    await element(by.text('Device Tap')).tap();
+    await device.tap();
+    await expect(element(by.text('Screen Tapped'))).toBeVisible();
+  });
+
+  it(':android: tap on screen by coordinates', async () => {
+    await device.launchApp({ newInstance: true });
+    await element(by.text('Device Tap')).tap();
+    const point = {x: 200, y: 243}
+    await device.tap(point);
+    await expect(element(by.text('Button Tapped'))).toBeVisible();
+  });
+
+  it(':android: long press on screen by coordinates', async () => {
+    await device.launchApp({ newInstance: true });
+    await element(by.text('Device Tap')).tap();
+    const point = {x: 150, y: 300}
+    await device.longPress(point);
+    await expect(element(by.text('Screen Long Pressed'))).toBeVisible();
+  });
+
+  it(':android: long press on screen by coordinates with duration', async () => {
+    await device.launchApp({ newInstance: true });
+    await element(by.text('Device Tap')).tap();
+    const point = {x: 200, y: 190}
+    await device.longPress(point, 2000);
+    await expect(element(by.text('Screen Long Custom Duration Pressed'))).toBeVisible();
+  });
```

Comment:
let's extract these to a new suite (e.g. `06.device-tap.test.js`)

Reply:
also add e2e tests for long-press with (1) duration only and (2) no params

Reply:
I would also add more tests:
- `should long press on specified point` (expect for **specific area** to be pressed when passing the `point` param)
- `should not succeed in long pressing with point outside the target area`

Check how we added a similar tests on `03.actions.test.js`.

Reply:
Same for `device.tap(point)`

Reply:
about the spetific point test these are `tap on button by coordinates ignoring status bar` and `long press on button by coordinates with duration ignoring status bar`. about failing on press outside the are these are `long press on screen with duration without specifying coordinates` and `tap on screen without specifying coordinates`
tell me if It is what you meant